Increase maximum number of mfns for XEN_DOMCTL_memory_mapping

The overhead of a complete data cache flush per 64 pages results
in a mapping/unmapping a 1GB taking 30 seconds which cause's test
timeouts.
diff --git a/xen/common/domctl.c b/xen/common/domctl.c
index 4ebb06f..03dba0f 100644
--- a/xen/common/domctl.c
+++ b/xen/common/domctl.c
@@ -1035,7 +1035,7 @@ long do_domctl(XEN_GUEST_HANDLE_PARAM(xen_domctl_t) u_domctl)
 
         ret = -E2BIG;
         /* Must break hypercall up as this could take a while. */
-        if ( nr_mfns > 64 )
+        if ( nr_mfns > 1024 )
             break;
 
         ret = -EPERM;
